terraform/context_import_test.go

```diff
@@ -227,6 +227,8 @@ func TestContextImport_moduleProvider(t *testing.T) {
 }
 
 // Test that import sets up the graph properly for provider inheritance
+// FIXME: import must declare a provider in an empty config. Should that go
+// back to being automatically inherited?
```

Comment:
We currently have a number of weird quirks with importing anywhere other than the root with a static config, unfortunately.

Our current state is that you must write at least a stub config to do a successful import, which _today_ means just a `resource` block but it seems reasonable that a `provider` block would also be required now in situations where it would be required for non-import operations, since otherwise we are likely to end up doing the import via the "wrong" provider, causing problems later when the imported resource may appear to be missing.